docs/test-specifications/token-service/TokenClaimTransaction.md

```diff
@@ -0,0 +1,140 @@
+| 8       | Claims a non-airdropped fungible token for an account                                                             | accountId=<CREATED_ACCOUNT_ID>, tokenId=<CREATED_TOKEN_ID>, commonTransactionParams.signers=[<CREATED_ACCOUNT_PRIVATE_KEY>]                                                                                                                               | The claim fails with a NO_REMAINING_AIRDROPS response code from the network.                                                                                  | N                 |
+| 9       | Claims an airdropped fungible token for an account without signing                                                | accountId=<CREATED_ACCOUNT_ID>, tokenId=<CREATED_TOKEN_ID>                                                                                                                                                                                               | The claim fails with an INVALID_SIGNATURE response code from the network.                                                                                     | N                 |
+| 10      | Claims an airdropped fungible token for an account that is frozen for the token                                   | accountId=<CREATED_ACCOUNT_ID>, tokenId=<CREATED_TOKEN_ID>, commonTransactionParams.signers=[<CREATED_ACCOUNT_PRIVATE_KEY>]                                                                                                                               | The claim fails with an ACCOUNT_FROZEN_FOR_TOKEN response code from the network.                                                                              | N                 |
+| 11      | Claims an airdropped paused fungible token for an account                                                         | accountId=<CREATED_ACCOUNT_ID>, tokenId=<CREATED_TOKEN_ID>, commonTransactionParams.signers=[<CREATED_ACCOUNT_PRIVATE_KEY>]                                                                                                                               | The claim fails with a TOKEN_IS_PAUSED response code from the network.                                                                                        | N                 |
+| 12      | Claims multiple airdropped fungible tokens for an account                                                         | accountId=<CREATED_ACCOUNT_ID>, tokenId=<CREATED_TOKEN_ID>, commonTransactionParams.signers=[<CREATED_ACCOUNT_PRIVATE_KEY>]                                                                                                                               | The claim succeeds and the account has all airdrops of the token credited to its balance.                                                                     | N                 |
```

Comment:
It looks like this test only specifies one token? I think maybe the inputs to this function need to be looked at to allow for multiple airdrops to be added to the request

Reply:
Ah I get it now, understood!